HOMEWORK_11/EX3.CPP

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+
+#define ROWS 4
+#define COLUMNS 4
+
+void from14to41(int arr_2d[ROWS][COLUMNS])
+{
+    for(int i = COLUMNS - 1; i >= 0; i--)
+    {
+        for(int j = 0; j < ROWS; j++)
+        {
+            std::cout<<arr_2d[j][i]<<" ";
+        }
+        std::cout<<std::endl;
+    }
+}
+
+void from41to11(int arr_2d[ROWS][COLUMNS])
+{
+    int turn = 2;
```

Comment:
bool variable would be enough
bool toRight = true;
if (toRIght) ... else ...
toRight = !toRight;